lib/assigner/include/bytecode.hpp

```diff
@@ -22,6 +22,8 @@
 namespace nil {
     namespace blueprint {
 
+        constexpr static size_t BYTECODE_TABLE_INDEX = 0;
```

Comment:
may be keep all circuit indexes in one place (assigner.hpp). I guess it's more convenient if we'll change order

Reply:
Yep, that was my initial idea, but the problem is that `assigner.hpp` depends on `bytecode.hpp` and `rw.hpp`, so we cannot include assigner in these files. Maybe create a separate file for these constants?

Reply:
may be pass table index as argument `process_bytecode_input` function?

Reply:
and type of index may be uint8_t, i guess 256 tables more then enough)

Reply:
Thanks, passed a specific table now. Kept the type as `size_t`, because the number will be converted to it anyway on passing into `std::vector<>::operator[]`. And I don't like the idea of using `uint8_t` when it's not decreases memory usage significantly. It is in fact `unsigned char`, so you have to convert it into `int` during any debugging actions to not get a character printed.